utils_nlp/models/transformers/named_entity_recognition.py

```diff
@@ -187,6 +195,10 @@ def _is_iterable_but_not_string(obj):
             )
             max_len = MAX_SEQ_LEN
 
+        logging.warn(
```

Comment:
Does this function only work for bert? If it works for other models, the name is misleading.